data/dataset_neo.py

```diff
@@ -40,20 +42,183 @@ def __init__(
         self.db = Database(db_param[0], db_param[1], db_param[2])
         self.split_type = split_type
 
+        self.default_edge_types = [Constants.edge_key]
+        self.other_edge_types = [Constants.edge_key_extra]
```

Comment:
I'm not sure if the unit tests work if these are in Constants. for unit tests, we pass in a custom config, so we can decouple its configuration from the main pipeline's. But we can't pass in Constants, as they're ... constants :D

Reply:
I have added these to config!